src/ProgressBar.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Bulma;
+
+use Yiisoft\Html\Html;
+
+/**
+ * Progress Bar widget
+ * Native HTML progress bar
+ * 
+ * ```php
+ * echo ProgressBar::widget()->progressValue(75);
+ * ```
+ * 
+ * @link https://bulma.io/documentation/elements/progress/
+ */
+final class ProgressBar extends Widget
+{
+    private array $options = [];
+    private float $progressValue = 0;
```

Comment:
Should be `float|null`. Null should not render a value and be a default. See below.